opentera-webrtc-native-client/OpenteraWebrtcNativeClient/python/package/setup.py.in

```diff
@@ -3,39 +3,43 @@ from setuptools.command.install import install
 import os
 
 
-with open("README.md", "r", encoding="utf-8") as fh:
-    long_description = fh.read()
-    
-    
-# force setuptools to recognize that this is
-# actually a binary distribution
+def read_file(name):
+    with open(name, "r", encoding="utf-8") as fh:
+        return fh.read()
+
+
+long_description = read_file("README.md")
+
+
 class BinaryDistribution(dist.Distribution):
+    """Force setuptools to recognize that this is actually a binary distribution"""
     def has_ext_modules(foo):
-        return True    
+        return True
+
 
 setup(
-    name="opentera_webrtc_native_client", 
+    name="opentera_webrtc_native_client",
     version="${OPENTERA_WEBRTC_NATIVE_CLIENT_VERSION}",
     author="Marc-Antoine Maheux, Cedric Godin, Dominic Letourneau",
     author_email="marc-antoine.maheux@usherbrooke.ca, cedric.godin@usherbrooke.ca, dominic.letourneau@usherbrooke.ca",
     description="OpenTera WebRTC Client Library for Python",
     long_description=long_description,
     long_description_content_type="text/markdown",
     url="https://github.com/introlab/opentera-webrtc",
-    packages=['opentera_webrtc_native_client'],
+    packages=['opentera.webrtc.native_client'],
```

Comment:
Moved to namespace package `opentera.webrtc`.
We might have to move the OpenTera Server to something like `opentera.server` instead of `opentera` as it is currently, as this would create conflicts

Reply:
This is a good idea!

Reply:
@doumdi 
Do you like the idea?